tools/icon-component-generator/util/fixLint.js

```diff
@@ -0,0 +1,17 @@
+// const standard = require('standard')
+
+module.exports = function fixLint (data) {
+  // TODO: support ts standard as straight js standard errors
+  return Promise.resolve(data)
+  // return new Promise((resolve, reject) => standard.lintText(data, { fix: true }, (err, standardData) => {
```

Comment:
do we want to keep all this commented out?

Reply:
yeah I think it's ok for now this is how it was done for browser-laptop which used standard, and I don't want to lose the flow since it's not pushed in any other repo.

For now we don't need to `lint --fix` since I made the template pass this project's linter.

Reply:
ok sounds good